### DISCUSSION TITLE
Mouse drifts

### DISCUSSION THREAD

**Discussion by ShiftingKill** [mgth/LittleBigMouse#281] [Q&A]
Settings:
https://pastebin.com/zKyV5iF7

Mouse drifts when making circles through 3 monitors. What I mean is, when making circular motions to hit each screen with the cursor, the transitions between screens are perfectly lined up, but I will have to keep moving/drifting my hand to the right in order to keep the circle hitting all 3 monitors. Not a big issue, but noticed it when I got bored and was making circles with the mouse. The cursor drifts left, so I compensate by moving my hand to the right.

**Comment by syborg64** [comment]
Probably has to do with your cursor speed, if the opposite behavior happens when making circles in in the other direction then it would confirm it. Try enabling correct cursor speed

**Comment by mgth** [comment]
That is because of border size, when you have tree monitors there must be a side where you instantly cross two borders when the other side you just move inside the display, making the travel longer.
